editor/components/inner-blocks/index.js

```diff
@@ -22,23 +22,23 @@ import BlockList from '../block-list';
 import { withBlockEditContext } from '../block-edit/context';
 
 class InnerBlocks extends Component {
-	componentDidMount() {
+	constructor() {
+		super( ...arguments );
+
 		this.updateNestedSettings();
```

Comment:
I don't really like that we rely on specific component lifecycle to effect nested locking.